blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_5_structural_analysis/formula_5_2.py

```diff
@@ -13,7 +14,7 @@ class Form5Dot2Eccentricity(Formula):
 
     def __init__(
         self,
-        theta_i: float,
+        theta_i: DIMENSIONLESS,
         l_0: M,
```

Comment:
I would call this `effective length` --> single letter 'I' and 'l' should be avoided : https://peps.python.org/pep-0008/#names-to-avoid

i could read this as: i_zero/Oh, el_zero/Oh or even one_zero (I know isn't possible as variable name)

Reply:
Too bad for me I couldnt participate 😢 I'm afraid I indeed wouldn't agree on this style 🤣 but I wil stop whining and adhere to it. thats the consequence of my own absence.